### PR TITLE
Added most content for day one, two and three

### PR DESCRIPTION
**Pull request by raphamontenegro**
- [x] Added schedule of the day, resources and exercises for the first three days.
- [x] Added homework for day one.
- [ ] homework for day two and three
- [ ] content for day four

### CODE REVIEW

File: _homework/web-development/portfolio-creation.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: Create your own webpage
+assigned: Oct 1, 2018
+due: Oct 2, 2018
+---
+
+
+Tasks
+--------
+
+- Fork Rapha's [repository](https://github.com/raphamontenegro)
```
Comment (nevanscott) [inline _homework/web-development/portfolio-creation.md:11]: At first glance, it doesn't seem like this link is a repo, just your main page. Should this link somewhere else?

**Review by nevanscott — CHANGES_REQUESTED** [review]
All the lessons look great!

For the homework, I noticed a link that might need to be updated before you publish. I'll let you double-check it.

File: _homework/web-development/portfolio-creation.md (hunk 1 above)
Comment (raphamontenegro) [reply _homework/web-development/portfolio-creation.md:11]: I'll create this repository tomorrow in class with them with the quick demo.  For now it is just a placeholder.

Comment (nevanscott) [reply _homework/web-development/portfolio-creation.md:11]: OK cool 👍

**Review by nevanscott — APPROVED** [review]

**Pull request merged**